indico/modules/events/timetable/controllers/manage.py

```diff
@@ -34,9 +35,16 @@ class RHManageTimetable(RHManageTimetableBase):
     session_management_level = SessionManagementLevel.coordinate
 
     def _process(self):
+        from indico.modules.events.contributions import contribution_settings
+
         event_info = serialize_event_info(self.event)
         timetable_data = TimetableSerializer(self.event, management=True).serialize_timetable()
+        show_draft_warning = (self.event.type_ == EventType.conference and
+                              not contribution_settings.get(self.event, 'published') and
+                              (TimetableEntry.query.with_parent(self.event).has_rows() or
+                               Contribution.query.with_parent(self.event).has_rows()))
```

Comment:
We have this in two places now. Maybe add a `should_show_draft_warning` util to `indico/modules/events/util.py` and call it in both places?

Reply:
Added :+1: